calico/etcddriver/driver.py

```diff
@@ -576,6 +578,13 @@ def _stop_watcher(self):
             self._watcher_stop_event = None
 
     def get_etcd_connection(self):
+        if self._etcd_url_parts.scheme == "https":
+            return HTTPSConnectionPool(self._etcd_url_parts.hostname,
+                                       self._etcd_url_parts.port or 2379,
+                                       key_file=self._etcd_key_file,
+                                       cert_file=self._etcd_cert_file,
+                                       ca_certs=self._etcd_ca_file,
+                                       maxsize=1)
         return HTTPConnectionPool(self._etcd_url_parts.hostname,
```

Comment:
Nitpick: I'd probably use an else: here for symmetry but that's totally aesthetic.